patterns/direct_requestor_blocking.py

```diff
@@ -0,0 +1,121 @@
+## Goal: A simpe request pulblisher to publish a request with reply_to set to any topic.
```

Comment:
Typo: simple, publisher

Reply:
I would also mention that this is a blocking requester

Reply:
Removed that block of comments - just sticking with convention fo 1-liner at the top of the file like other samples.